Makefile

```diff
@@ -95,4 +95,4 @@ test-e2e: image hyperkube
 		-chyperkube_image=$(HYPERKUBE_IMAGE)
 
 test-ccm-e2e:
-	go test ./tests/e2e/
+	go test ./tests/e2e/ -timeout 0
```

Comment:
so what's the timeout value here?

Reply:
`0` disables the default 10m timeout. As we run this more in the future, we will get a sense of what timeout value we should set here.

Reply:
Reasonable